torch/testing/_internal/common_device_type.py

```diff
@@ -1231,6 +1239,12 @@ def onlyCUDA(fn):
 def onlyMPS(fn):
     return onlyOn('mps')(fn)
 
+def onlyPRIVATEUSE1(fn):
+    device_type = torch._C._get_privateuse1_backend_name()
+    device_mod = getattr(torch, device_type, None)
+    assert device_mod is not None, f'''torch has no module of `{self.device_type}`, you should register
```

Comment:
I feel like this function should work even when there is no private namespace available: it should just skip the test.

Reply:
Yeah，I‘ve fixed it.